### ISSUE TITLE
Unable to upload .dashboadaddin file to NSEC Portal

### ISSUE THREAD

**Issue by ACueva** [Esri/ops-server-config#734]
As a non-administrator to the NSEC server, you are unable to upload .dashboardaddin files.
![addin issue](https://cloud.githubusercontent.com/assets/3093940/6538573/d1144814-c418-11e4-80bd-586dcedf2bb4.png)

Even if you are a Portal for ArcGIS administrator you are unable to upload these type of files unless you are also an administrator on the NSEC server.

**Comment by ACueva** [comment]
@conklinbd Portal Alias was contacted regarding this issue.  Did not get any feedback.  Is there someone specific I should ping regarding this issue?

**Comment by conklinbd** [comment]
Yes there is. @elinz  can point you in the right direction to the Product Engineer on the team. I just don't want to throw their email up here.

**Comment by elinz** [comment]
@ACueva I'll send you an email with two people you can contact.

**Comment by ACueva** [comment]
@conklinbd I've only been able to reproduce this issue on our development server.  I've tested against 3 of the portal team's QA servers and have not been able to reproduce this issue.  The work-around is either have an admin upload the item for the user, grant the user admin rights to the development server, or use a built-in account.  At this time I'm thinking this is just a fluke on the build we are currently using and/or some unknown elevated permissions we have set on the machine.  My recommendation is to keep an eye on this issue for the next build of Portal for ArcGIS.
